pages/api/trips/index.js

```diff
@@ -7,7 +7,16 @@ export default async function handler(request, response) {
   if (request.method === "GET") {
     const trips = await Trip.find();
     return response.status(200).json(trips);
-  } else {
-    return response.status(405).json({ message: "Method not allowed" });
+  }
+
+  if (request.method === "POST") {
+    try {
+      const tripData = request.body;
+      await Trip.create(tripData);
+
+      response.status(201).json({ status: "Trip created" });
+    } catch (error) {
+      response.status(400).json({ message: error.message });
+    }
   }
```

Comment:
I suggest to include an error message, when a method other than `GET` and `POST` is used:

```suggestion
  }
      return response.status(405).json({ message: "Method not allowed" });

```